File: C++/URI_1018.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    long int N;
    cin >> N;
    cout << N << endl;
    int n100 = N/100;
    N -= n100 * 100;
    
    int n50 = N/50;
    N -= n50 * 50;
    
    int n20 = N/20;
    N -= n20 * 20;
    
    int n10 = N/10;
    N -= n10 * 10;
    
    int n5 = N/5;
    N -= n5 * 5;
    
    int n2 = N/2;
    N -= n2 * 2;
    
    int n1 = N/1;
    
    cout << n100 << " nota(s) de R$ 100,00\n";
    cout << n50 << " nota(s) de R$ 50,00\n";
    cout << n20 << " nota(s) de R$ 20,00\n";
    cout << n10 << " nota(s) de R$ 10,00\n";
    cout << n5 << " nota(s) de R$ 5,00\n";
    cout << n2 << " nota(s) de R$ 2,00\n";
    cout << n1 << " nota(s) de R$ 1,00\n";
    
 
    return 0;
}

File: Python/URI_1018.py
valor = int(input())

print(valor)

n100 = valor // 100
valor = valor%100

n50 = valor // 50
valor = valor%50

n20 = valor // 20
valor = valor%20

n10 = valor // 10
valor = valor%10

n5 = valor // 5
valor = valor%5

n2 = valor // 2
valor = valor%2

n1 = valor // 1
valor = valor%1


print(f'{n100} nota(s) de R$ 100,00')
print(f'{n50} nota(s) de R$ 50,00')
print(f'{n20} nota(s) de R$ 20,00')        
print(f'{n10} nota(s) de R$ 10,00')        
print(f'{n5} nota(s) de R$ 5,00')  
print(f'{n2} nota(s) de R$ 2,00')
print(f'{n1} nota(s) de R$ 1,00') 